drake/bindings/swig/autodiffutils.i

```diff
@@ -0,0 +1,39 @@
+%module(package="pydrake") autodiffutils
+
+
+%include "exception_helper.i"
+%include <eigen.i>
+%include <autodiff.i>
+
+#ifdef SWIGPYTHON3
+%rename(__truediv__)     *::operator/;
+#endif
+
+%template(AutoDiffVectorDynamic) AutoDiffWrapper<Eigen::VectorXd, Eigen::Dynamic, 1>;
+%template(AutoDiffVectorMax73) AutoDiffWrapper<Eigen::Matrix<double, Eigen::Dynamic, 1, 0, 73>, Eigen::Dynamic, 1>;
+%template(AutoDiffMatrixDynamic) AutoDiffWrapper<Eigen::VectorXd, Eigen::Dynamic, Eigen::Dynamic>;
```

Comment:
Implementation is here: https://github.com/rdeits/swigmake/blob/master/swig/common/autodiff.i